Makefile

```diff
@@ -15,18 +15,20 @@
 # Set ENV to 'YELP' if FQDN ends in '.yelpcorp.com'
 # Otherwise, set ENV to the FQDN
 ifeq ($(findstring .yelpcorp.com,$(shell hostname -f)), .yelpcorp.com)
```

Comment:
Sounds fine to me. I'll merge this for now since it doesn't change the status quo (laptops were using public PyPI before) but happy to have it also check an environment variable. Can you suggest a name? (Just `YELP`?)